.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules
 .env*
 !.env.example
-logs
\ No newline at end of file
+logs
+credit-cards.json
```

Comment:
@bryank91 can you pull the latest master? `credit-cards.json` was intentionally removed from the `.gitignore` and adding to tracking.

Reply:
@samc621  Can you review this? I've removed this as its not necessary.